src/components/datepicker/demoBasicUsage/index.html

```diff
@@ -10,17 +10,26 @@
     <h4>Date-picker with min date and max date</h4>
     <md-datepicker ng-model="myDate" md-placeholder="Enter date"
         md-min-date="minDate" md-max-date="maxDate"></md-datepicker>
-
+    <h4>Only weekends are selectable</h4>
+    <md-datepicker ng-model="myDate" md-placeholder="Enter date"
+        md-date-enabled="onlyWeekendsPredicate"></md-datepicker>
+    
+    <h4>Only weekends within given range are selectable</h4>
+    <md-datepicker ng-model="myDate" md-placeholder="Enter date"
+        md-min-date="minDate" md-max-date="maxDate" md-date-enabled="onlyWeekendsPredicate"></md-datepicker>
+    
     <h4>With ngMessages</h4>
     <form name="myForm">
       <md-datepicker name="dateField" ng-model="myDate" md-placeholder="Enter date"
-          required md-min-date="minDate" md-max-date="maxDate"></md-datepicker>
+          required md-min-date="minDate" md-max-date="maxDate" md-date-enabled="onlyWeekendsPredicate"></md-datepicker>
 
       <div class="validation-messages" ng-messages="myForm.dateField.$error">
         <div ng-message="required">This date is required!</div>
         <div ng-message="mindate">Date is too early!</div>
         <div ng-message="maxdate">Date is too late!</div>
+        <div ng-message="disabled">Date is disabled!</div>
```

Comment:
If attribute is changed to md-date-filter, I guess error message key should be changed to 'filtered' instead of disabled?